scripts/vim-patch.sh

```diff
@@ -886,7 +887,41 @@ review_pr() {
   clean_files
 }
 
-while getopts "hlLmMVp:P:g:r:s" opt; do
+is_na_patch() {
```

Comment:
Can add a new flag to only check 1 patch
but it may have more value in its own function to return the list of relevant files.